spec/spec.md

```diff
@@ -102,6 +103,82 @@ _reflector_ - an intermediary which brokers connections between two endpoints, w
 
 _wallet endpoint_ - an app implementing wallet-like functionality (i.e. providing transaction signing services). This endpoint acts as the server in this protocol.
 
+## Identifiers
+
+Namespaced identifiers are used in the format `${namespace}:${reference}` to refer to objects like blockchains and features in a canonical and human readable form. 
+
+### Chain Identifiers
+
+Namespaced chain identifiers are used to refer to blockchains with which a dapp intends to interact with. This scheme is compatible with [CAIP-2](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md) chain IDs, but are not required to be used.
+
+The currently supported Solana network chains are `solana:mainnet`, `solana:testnet`, and `solana:devnet`. 
+
+Previous versions of this specification used a `cluster` parameter to specify the Solana network cluster with which the dapp endpoint intends to interact. The supported values for the `cluster` parameter were `mainnet-beta`, `testnet`, and `devnet`. These values have been replaced by the above chain references.
+
+### Feature Identifiers
+
+Feature identifiers are used to identify features that are supported by a wallet or account. The currently defined standard features include: 
+
+- `solana:signMessages`
+- `solana:signTransactions`
+- `solana:signAndSendTransaction`
+- `solana:signInWithSolana`
+
+#### Feature Descriptions
```

Comment:
This section is TBD, and where I need the most help/input.

How should we describe feature interfaces in a language agnostic way? Here I have drafted a spec using JSON RPC method descriptions similar to how we are already describing methods in the Wallet RPC Interface section.  Questions:

1. Should features be versioned? this would allow us to increment specific features/methods without updating the entire spec. 
2. How should params, results and errors be described? Should we simply provide the names for these and expect endpoints to consult relevant documentation for the exact types and formats of these fields? Or should feature descriptions include complete documentation of the fields including their types, descriptions, etc? We could use an existing IDL standard for this, such as [Apache Avro](https://avro.apache.org/docs/1.11.1/specification/), or define our own. 

Motivation:
We want wallet endpoints to be able to describe their capabilities in a way that allows folks to extend the spec without requiring an updated version of the core specification, similar to how the wallet-standard allows new and experimental features to be defined and described by the [`Wallet` Interface](https://github.com/wallet-standard/wallet-standard/blob/ce280f05f6eb16d0e77939827e0b7fa294460e1f/packages/core/base/src/wallet.ts#L33)